StarRailBattleSim/src/relics/relics/PioneerDiverOfDeadWaters.java

```diff
@@ -0,0 +1,67 @@
+package relics.relics;
+
+import characters.AbstractCharacter;
+import enemies.AbstractEnemy;
+import powers.PermPower;
+import powers.PowerStat;
+import relics.AbstractRelicSetBonus;
+
+import java.util.ArrayList;
+
+/**
+ * The 4PC 100% boost after inflicting a debug is not working currently
+ */
+public class PioneerDiverOfDeadWaters extends AbstractRelicSetBonus {
+    public PioneerDiverOfDeadWaters(AbstractCharacter owner, boolean fullSet) {
+        super(owner, fullSet);
+    }
+
+    public PioneerDiverOfDeadWaters(AbstractCharacter owner) {
+        super(owner);
+    }
+
+    @Override
+    public void onEquip() {
+        this.owner.addPower(new PioneerDiverOfDeadWaters2PC());
+
+        if (this.isFullSet) {
+            this.owner.addPower(new PioneerDiverOfDeadWaters4PC());
+        }
+    }
+
+    public static class PioneerDiverOfDeadWaters2PC extends PermPower {
+        public PioneerDiverOfDeadWaters2PC() {
+            this.name = this.getClass().getSimpleName();
+        }
+
+        @Override
+        public float getConditionalDamageBonus(AbstractCharacter character, AbstractEnemy enemy, ArrayList<AbstractCharacter.DamageType> damageTypes) {
+            if (enemy.powerList.stream().anyMatch(p -> p.type == PowerType.DEBUFF)) {
+                return 12;
+            }
+
+            return 0;
+        }
+    }
+
+    public static class PioneerDiverOfDeadWaters4PC extends PermPower {
+
+        private boolean boosted = false;
+
+        public PioneerDiverOfDeadWaters4PC() {
+            this.name = this.getClass().getSimpleName();
+            this.setStat(PowerStat.CRIT_CHANCE, 4);
```

Comment:
the crit chance bonus is also doubled when the wearer applies a debuff, just FYI

Reply:
Well, my acheron has too much CR then